packages/standard-linter/index.js

```diff
@@ -11,6 +11,13 @@ module.exports = {
   rules: {
     "@typescript-eslint/no-floating-promises": "error",
     "no-console": "error",
+    "max-classes-per-file": "off",
+    "class-methods-use-this": "off",
+    "no-await-in-loop": "off",
+    // functions and classes are going to be hoisted in runtime
+    "no-use-before-define": ["error", { functions: false, classes: false }],
+    // https://github.com/airbnb/javascript/issues/1271
+    "no-restricted-syntax": "off",
```

Comment:
will this allow `for-of` syntax? thought `forEach` would be better because of FP and maintainability aspect :o

Reply:
and we can always eslint-disable-next-line if needed (though usually that means the code can be simplified into a normal forEach loop?)

Reply:
I think there's a reason by `for-of` is still in the js spec. I personally don't use `for of` that often but there is still benefit of `for-of` over `forEach`, like how you can't break `forEach` early unlike `for-of` you can do standard loop control flows like `break`, `continue` and etc.

Reply:
ok, as discussed with @Nanosync will remove this eslint rule and let user turn off eslint by comment if necessary

Reply:
Oops forgot to add this also allows the other disabled syntaxes (with, labels for goto)